tests/test_web_urldispatcher.py

```diff
@@ -515,3 +516,17 @@ async def test_static_absolute_url(aiohttp_client, tmpdir) -> None:
     client = await aiohttp_client(app)
     resp = await client.get("/static/" + str(fname))
     assert resp.status == 403
+
+
+async def test_decoded_match_regex(aiohttp_client) -> None:
```

Comment:
Once that other test I requested below is added and it's recorded in the CI logs, please apply an xfail marker so we could incorporate this in master:
```suggestion
@pytest.mark.xfail(
    raises=AssertionError,
    reason="Regression in v3.7: https://github.com/aio-libs/aiohttp/issues/5621",
)
async def test_decoded_match_regex(aiohttp_client) -> None:
```

Reply:
done